jekyll/_cci2/enable-checks.md

```diff
@@ -53,6 +53,11 @@ To disable the CircleCI Check integration, navigate to the Org Settings Page, th
 2. Select VCS. 
 3. Click the Manage GitHub Checks button. The Update CircleCI Checks repository access page appears. ![CircleCI VCS Settings Page]( {{ site.baseurl }}/assets/img/docs/checks_update.png)
 4. Deselect the repository to uninstall the Checks integration.
+5. Refresh the status settings on your projects. Go to CircleCI > Project
+   Settings > Advanced Settings > Toggle the setting "Github Status Updates" off
```

Comment:
When I did this previously I did not need to flip back and forth, it was set to Off and I just set it to On, but that was months back. Let me validate quick.

Reply:
OK, depends on how you uninstall checks from GH. If you remove it from a single repo, the repo will be left "off" - if you "uninstall" it seems to revert status back to on.

But instead of saying "toggle off/on" maybe just "confirm the Github Status Updates is on"

Reply:
All set! Thanks for the review!